javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Hack Your Future, Javascript, homework Week2
+ * this file includes:
+   Student manager program for adding up to 6students to a class.
+*/
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "Queen") {
+        class07Students.push(studentName);
+        return 'We always have a space for our Queen';
+    } else if (!studentName) {
+        return 'Please write student name.';
+    } else if (duplicateCheck(studentName)) {
+        return 'We already have ' + studentName + ' in the class 07.';
+    } else if (getNumberOfStudents() > 5) {
```

Comment:
Maybe add this as `const numberOfStudents=5`

It's better in general to name your constants in the code

Reply:
Hi Florin, I didn't get that what should I do here. Could you please explain it more.

Reply:
HI Florin, I didn't get what should I change here. Could you please explain it more